WIP-CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 - The `clusterDns` config variable now matches Kubespray defaults.
   Using the wrong value causes node-local-dns to not be used.
+- Grafana in the kube-prometheus-stack are upgraded to fix dashboard errors.
```

Comment:
You should change this to instead mention that kube-prometheus-stack are upgraded